### PR TITLE
Change relative links to absolute links

### PR DESCRIPTION
**Pull request by fosslinux**
Change some relative links to absolute links, as on linuxbrew.sh, these links are dead as they point to 404s.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 The Homebrew package manager may be used on Linux and Windows 10, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about). Homebrew is referred to as Linuxbrew when running on Linux or Windows. It can be installed in your home directory, in which case it does not use *sudo*. Linuxbrew does not use any libraries provided by your host system, except *glibc* and *gcc* if they are new enough. Linuxbrew can install its own current versions of *glibc* and *gcc* for older distribution of Linux.
 
-[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](Formula-Cookbook.md#homebrew-terminology).
+[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](https://github.com/Linuxbrew/brew/blob/master/docs/Formula-Cookbook.md#homebrew-terminology).
```
Comment (sjackman) [inline index.md]: Thanks! Please change this link to here: https://docs.brew.sh/Formula-Cookbook.html#homebrew-terminology

File: index.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Second, read the [Troubleshooting Checklist](https://docs.brew.sh/Troubleshootin
 
 ## Contributing
 
-We'd love you to contribute to Linuxbrew. First, please read our [Contribution Guide](CONTRIBUTING.md) and [Code of Conduct](CODE_OF_CONDUCT.md#code-of-conduct). Please see our [guidelines](CONTRIBUTING.md#contributing-to-linuxbrew) on whether to send pull requests to Linuxbrew or Homebrew.
+We'd love you to contribute to Linuxbrew. First, please read our [Contribution Guide](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODE_OF_CONDUCT.md#code-of-conduct). Please see our [guidelines](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew) on whether to send pull requests to Linuxbrew or Homebrew.
```
Comment (sjackman) [inline index.md:102]: 👍

File: index.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ Former maintainers with significant contributions include [JCount](https://githu
 - [Linuxbrew category](https://discourse.brew.sh/c/linuxbrew) of [Homebrew's Discourse](https://discourse.brew.sh)
 
 ## License
-Code is under the [BSD 2-clause "Simplified" License](LICENSE.txt).
+Code is under the [BSD 2-clause "Simplified" License](https://github.com/Linuxbrew/brew/blob/master/LICENSE.txt).
```
Comment (sjackman) [inline index.md:145]: 👍

**Review by sjackman — CHANGES_REQUESTED** [review]
One small change. Thanks, @fosslinux!

**Comment by fosslinux** [comment]
@sjackman sorry that took so long, i think its good now!

File: index.md (hunk 1 above)
Comment (fosslinux) [reply index.md]: fixed

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
No worries at all. Merged. Thanks for your contribution to Linuxbrew, @fosslinux! 🏅

**Pull request merged**